lldb/source/Plugins/Process/AIX/NativeProcessAIX.cpp

```diff
@@ -0,0 +1,248 @@
+//===-- NativeProcessAIX.cpp --------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "NativeProcessAIX.h"
+#include "NativeThreadAIX.h"
```

Comment:
I did test everything while pushing, must be a blunder thats still there. Apologies

Reply:
One point though, linux wont build these files and on AIX we are not currently at a state where it will build anyway since there is not enough AIX related code. So what can we try?

Reply:
What's stopping it from building? I'd definitely prioritize those changes, otherwise this whole incremental review process is just an illusion. For example, if the build fails to to a lack of HostInfo class, then I'd suggest adding the most minimal HostInfoAIX class you can create. It doesn't have to have all the bells and whistles that the linux version does (as the main point of contention is whether those are linux-specific bells or not). I'd happily approve an empty HostInfo class to get the build working.

Reply:
I think the base ones in that regard will be HostInfo and Platform plugins. We are already discussing on the Host part, I will create another one with the Platform changes although even that is Linux derived as well, so maybe a minimal one for that as well might be a good idea?

Reply:
Yes, sounds good. The platform class shouldn't be even used from this code, so a really barebone implementation should be sufficient to get the code building, and maybe even to get this code (lldb-server) operational.